client/dive-common/components/RunPipelineMenu.vue

```diff
@@ -69,13 +87,16 @@ export default defineComponent({
         });
         // Filter out unsupported pipelines based on subTypeList
         // measurement can only be operated on stereo subtypes
-        if (name === stereoPipelineMarker) {
-          if (props.subTypeList.length === props.subTypeList.filter((item) => item === 'stereo').length) {
+        if (props.subTypeList.length === props.subTypeList.filter((item) => item === 'stereo').length
+        && (name === stereoPipelineMarker)) {
+          sortedPipelines[name] = category;
+        } else if (props.subTypeList.length === props.subTypeList.filter((item) => item === 'multicam').length
+        && (multiCamPipelineMarkers.includes(name))) {
+          if (name.split('-')[0] === camNumber.value.toString()) {
```

Comment:
this gives me anxiety.  I don't know what to do about it though.

Reply:
could restructure `const multiCamPipelineMarkers = ['2-cam', '3-cam'];` to be an object with a key-name and the number of cameras it represents?